### DISCUSSION TITLE
Promotions – configure site / localization for name + description

### DISCUSSION THREAD

**Discussion by zackspear** [craftcms/commerce#2802] [Ideas]
We have a discount that will soon go live on our Commerce instance. Our content editor was asking for the discount name + description to be translatable but noticed that it's not possible as there's no site dropdown available in the Promotions settings.

While the majority of our users use our default English site we have four other languages – Chinese, French, German, & Spanish. So being able to display the discount / sale details in a site's language would be helpful for our non-English users – especially our Chinese users.

Thanks for the consideration 🙏

**Comment by WHITE-developer** [comment]
We've came across the same issue for our multi language shops. So we also would like to see these translatable. Thanks.
